templates/application-kube-prometheus-stack.yaml

```diff
@@ -91,6 +92,22 @@ spec:
                     requests:
                       storage: {{ .Values.prometheus.volume_claim_storage_request}}Gi
             hostNetwork: {{ .Values.host_network.enabled }}
+            replicaExternalLabelName: __replica__
+            prometheusExternalLabelName: cluster
```

Comment:
Do we need this `prometheusExternalLabelName` ?

Reply:
I'm assuming this just adds "cluster" to all the emitted metrics?

Reply:
`prometheusExternalLabelName: cluster`

makes "cluster" as the key for the label and for the value it picks it from 

```
externalLabels:  (this has not been added yet)
  cluster: tenant01
```

so cortex would identify the prometheus instances from the same cluster with the "cluster" key.

Now that we are going without HA, I guess this can be removed ?

ref: https://cortexmetrics.io/docs/guides/ha-pair-handling/#client-side

Reply:
Oh got it. Yeah let's just remove it since it sounds like it only benefits us in HA